misaligned.c

```diff
@@ -15,7 +15,8 @@ int printColorMap() {
 
 int main() {
     int result = printColorMap();
+    assert(result == 0);
     assert(result == 25);

```

Comment:
do you expect output to be both 0 and 25? is that possible?
Please add test cases to verify alignment of output strings.